modules/basics/src/test/java/com/opengamma/basics/date/DayCountTest.java

```diff
@@ -289,6 +289,24 @@ public void test_yearFraction(DayCount dayCount, int y1, int m1, int d1, int y2,
     assertEquals(dayCount.yearFraction(date1, date2), expected, 0d);
   }
 
+  @Test(dataProvider = "yearFraction")
+  public void test_relativeYearFraction(
+      DayCount dayCount, int y1, int m1, int d1, int y2, int m2, int d2, Double value) {
+    double expected = (value == SIMPLE_30_360 ? calc360(y1, m1, d1, y2, m2, d2) : value);
+    LocalDate date1 = LocalDate.of(y1, m1, d1);
+    LocalDate date2 = LocalDate.of(y2, m2, d2);
+    assertEquals(dayCount.relativeYearFraction(date1, date2), expected, 0d);
```

Comment:
It's a test. And you will confuse more people using a third argument to assertEquals when they expect 2. And when the 2-arg call matches what the 3-arg call does. i.e the code could be simpler, and simpler is good